activerecord/lib/active_record/core.rb

```diff
@@ -96,6 +96,15 @@ def self.error_on_ignored_order_or_limit=(value)
         self.error_on_ignored_order = value
       end
 
+      # :singleton-method:
+      # Specify the behavior for unsafe raw query methods. Values are as follows
+      #   enabled    - Unsafe raw SQL can be passed to query methods.
```

Comment:
As a security feature, we want to allow people to opt in to the new safe behaviour immediately, once their app has wrapped any calls that intended to pass SQL. And probably to make newly generated apps safe too.

I'm not sure we need to support a `disabled` mode though.. this can probably just be a boolean to choose between a raise and a deprecation warning.